docs/guides/src/main/server/logging.adoc

```diff
@@ -7,90 +7,100 @@ title="Configuring logging"
+=== Category-specific log level configuration
+You can set different log levels for specific areas in Keycloak. Use this command to provide a comma-separated list of categories for which you want a different log level:
 
 <@kc.start parameters="--log-level=<root-level>,<org.category1>:<org.category1-level>"/>
 
-A configuration that applies to a category also applies to all sub-categories of that category, unless a more specific matching sub-category configuration is provided in the list.
+A configuration that applies to a category also applies to its sub-categories unless you include a more specific matching sub-category.
 
 .Example
 <@kc.start parameters="--log-level=INFO,org.hibernate:debug,org.hibernate.hql.internal.ast:info"/>
-The example above sets the root log level for all loggers to INFO, and the hibernate log level in general to debug. But as we don't want SQL abstract syntax trees to make the log output verbose, we set the more specific sub category `org.hibernate.hql.internal.ast` to info, so the SQL abstract syntax trees, which would be shown at `debug` level, don't show up anymore.
 
-== Configuring the logging format
-Keycloak uses a pattern-based logging formatter that generates human-readable text logs by default.
+This example sets the following log levels:
 
-The logging format template for these lines can be applied at the root level.
+* Root log level for all loggers is set to INFO.
+* The hibernate log level in general is set to debug.
+* To keep SQL abstract syntax trees from creating verbose log output, the specific subcategory `org.hibernate.hql.internal.ast` is set to info. As a result, the SQL abstract syntax trees are omitted instead of appearing at the `debug` level.
 
-The default format template is:
+== Logging format configuration
+Keycloak uses a pattern-based logging formatter that generates human-readable text logs. The logging format template for these lines can be applied at the root level. The default format template is:
 
 * `%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n`
 
-The format string supports the following symbols:
+The format string supports the symbols in the following table:
 
 |====
 |Symbol|Summary|Description
 |%%|%|Renders a simple % character.
 |%c|Category|Renders the log category name.
 |%d{xxx}|Date|Renders a date with the given date format string.String syntax defined by `java.text.SimpleDateFormat`
-|%e|Exception|Renders the thrown exception, if any.
+|%e|Exception|Renders any thrown exception that exists.
 |%h|Hostname|Renders the simple host name.
 |%H|Qualified host name|Renders the fully qualified hostname, which may be the same as the simple host name, depending on the OS configuration.
 |%i|Process ID|Renders the current process PID.
-|%m|Full Message|Renders the log message plus exception (if any).
+|%m|Full Message|Renders the log message plus any exception that exists.
```

Comment:
```suggestion
|%m|Full Message|Renders the log message and an exception, if thrown.
```

Reply:
see above. "any that exists" is not true. One line of log contains one exception. There may be more, e.g. when you set log to trace there are actually exceptions right now. These would not be printed, but "any...that exists" suggests so imo.